src/pages/patterns/fluid-variants/index.mdx

```diff
@@ -0,0 +1,200 @@
+50%) of the container’s width and will often be attached to the bottom of the
+containers. Fluid buttons cannot be embedded in the middle of a flow/layout.
+They must hug the (bottom) edge of a container. Use fixed buttons if additional
+options or actions follow below the primary flow and primary action.
+
+A fixed button’s width is calculated by the label’s length plus an additional
+64px to the right, with only the left side aligning to grid. Additionally, there
+is a hybrid scenario where a fixed styled button can span a designated number of
+columns on the responsive column grid giving it a fluid width. For more guidance
+on fixed versus fluid buttons, see the button component.
+
+### Fluid containers
+
+Though not a formal component, fluid and fixed containers are applied in various
+existing components. Fluid containers, like a modal, calculate their width based
+on column spans or percentages and will fluctuate in size. Fixed containers,
+like a side panel, maintain a static width, cannot be collapsed and can exist
+outside the responsive grid.
+
+### Fluid forms
+
+Fluid forms are architectural and remain cohesive by never allowing vertical or
+horizontal space between inputs. Fluid forms use the condensed or narrow grid
+mode and can hang into the gutter without causing label misalignment. Default
+forms use a wide grid mode and align flush to the grid columns prioritizing
+vertical label alignment.
+
+Fluid forms should only be used for simple, straightforward tasks, and any
+complex or multi-section forms should use the default style instead. For more
+guidance on fixed versus fluid forms, see the form component.
+
+## Designing with fluid
+
+### Visual guidance
+
+#### Alignment
+
+- Fluid component variants are stacked flush to one another with 0px of padding
+  between them.
+- Default component variants are equally stacked with padding separating each
+  component
+
+#### Sizing
+
+- Fluid variant’s width are dependent on its fluid container size or grid column
+  span. Fluid components can change per breakpoint or according to their defined
+  percentage.
+- Default variants may have fluid widths to a certain point but will commonly
+  have a max or min width to prevent it from continuously growing
+
+### Best practices
+
+- Fluid variants always appear in a container or as a structural unit. Default
+  variants may or may not appear within a container and margins will vary
+  according to its location on the grid.
+- Together, fluid variants comprise fluid containers, whereas default variants
+  can either sit on the page, without a container, or inside a container in with
+  inner margins.
+- Default and fluid variants can be used together in certain hybrid scenarios.
+  The most common is pairing default input components with fluid buttons in a
+  contained form, like in a modal or side panel.
+- Fluid variants use the condensed or narrow grid modes while default variants
```

Comment:
Fluid components use the condensed or narrow grid modes while default variants use the wide or narrow grid mode.